pkg/thing/mocks/thing_controller.go

```diff
@@ -0,0 +1,78 @@
+package mocks
+
+import (
+	"errors"
+
+	"github.com/stretchr/testify/mock"
+)
+
+// FakeController represents a mocking thing controller
+type FakeController struct {
+	mock.Mock
+}
+
+// Register provides a mock function to not return error
+func (f *FakeController) Register(body []byte, authorizationHeader string) error {
+	if len(body) == 0 {
+		return errors.New("empty body")
```

Comment:
Same here (and all other occurrences in this file), about error declarations :+1: